doc/subframes/subframes_tutorial.rst

```diff
@@ -0,0 +1,29 @@
+Subframes Tutorial
+============================
+
+This is just to test the subframes in a easy to run format.
```

Comment:
I'd be fine merging this tutorial as-is now (after addressing feedback), and improving it in the future when you have time. Please add note here that its a stub, though.

Reply:
Thanks for all the comments in a very rough file that wasn't meant to be in a PR yet :)
I would prefer to finish this up properly before merging it. Otherwise it won't really be displayable on the website at all. Give me a week or two until things cool down and I'll write up something proper.

Reply:
From many experiences, the statement "I'll get to it when things cool down" rarely happens, so even this would be better than nothing ;-)